elite_arm_driver/launch/bringup.launch.py

```diff
@@ -33,19 +34,20 @@ def execution_stage(context: LaunchContext):
     )
 
     rviz_launch = IncludeLaunchDescription(
-            PythonLaunchDescriptionSource([elite_description, '/elite_description.launch.py']),
+            PythonLaunchDescriptionSource(os.path.join(elite_description_pkg, 'launch', 'elite_description.launch.py')),
             condition=IfCondition(start_rviz)
-        )
+    )
 
-    urdf = os.path.join(get_package_share_directory('elite_description'), 'urdf', 'ec66_description_real.urdf')
+    robot_description_urdf = os.path.join(elite_description_pkg, 'urdf', 'ec66_description.urdf.xacro')
```

Comment:
when we launch the real robot, shouldn't you be setting the `use_gazebo` argument to `false` so that the Gazebo plugins are not loaded?

Reply:
no anna i have set it to default to false inside the urdf itself. So only when it is true we need to explicitly give it value. If u want I can explicitly include it here anna.